apps/docs/src/app/core/component-docs/message-toast/examples/message-toast-example.component.ts

```diff
@@ -8,28 +9,35 @@ import { MessageToastContentExampleComponent } from './message-toast-content-exa
     styleUrls: ['message-toast-example.component.scss']
 })
 export class MessageToastExampleComponent {
-    constructor(public messageToastService: MessageToastService) {}
+    messageToastText: string = null;
+
+    constructor(public messageToastService: MessageToastService, public liveAnnouncer: LiveAnnouncer) {}
 
     openFromComponent(): void {
         this.messageToastService.open(MessageToastContentExampleComponent, {
             mousePersist: true,
             duration: 5000,
             maxWidth: '25rem'
         });
+        this.messageToastText = this.messageToastService.textMessage;
+        this.liveAnnouncer.announce(this.messageToastText);
```

Comment:
That's a great improvement!

But shouldn't we call live announcer's method from the library component, not from an example? To don't push customers to write such code in their applications.

Reply:
done